### PR TITLE
UIMARCAUTH-305 Change tenant id to central when opening details of Shared Authority

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Change tenant id to central when opening details of Shared Authority

## Screenshots

https://github.com/folio-org/ui-marc-authorities/assets/19309423/27ac5297-047d-4dca-abec-63679201f8bf

## Issues
[UIMARCAUTH-305](https://issues.folio.org/browse/UIMARCAUTH-305)

## Related PRs
https://github.com/folio-org/ui-plugin-find-authority/pull/74
https://github.com/folio-org/stripes-authority-components/pull/99
https://github.com/folio-org/ui-quick-marc/pull/580

### CODE REVIEW

File: src/routes/AuthorityViewRoute/AuthorityViewRoute.test.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import { render } from '@folio/jest-config-stripes/testing-library/react';
 
-import { useMarcSource } from '@folio/stripes-authority-components';
+import {
+  useMarcSource,
+  useAuthority,
```
Comment (Dmytro-Melnyshyn) [inline src/routes/AuthorityViewRoute/AuthorityViewRoute.test.js:5]: Shouldn't all requests be with the central tenant id for the `shared` record upon opening the authority detail view, not just these two?

Comment (BogdanDenis) [reply src/routes/AuthorityViewRoute/AuthorityViewRoute.test.js:5]: You're right, created a new PR to set tenantId when fetching linking rules when viewing source record
https://github.com/folio-org/ui-quick-marc/pull/580

Comment (Dmytro-Melnyshyn) [reply src/routes/AuthorityViewRoute/AuthorityViewRoute.test.js:5]: Is it supposed to do the same for:
- mapping-rules/marc-authority
- authority-source-files
- linking-rules/instance-authority

Comment (Dmytro-Melnyshyn) [reply src/routes/AuthorityViewRoute/AuthorityViewRoute.test.js:5]: image

Comment (Dmytro-Melnyshyn) [reply src/routes/AuthorityViewRoute/AuthorityViewRoute.test.js:5]: It looks like not all requests leverage central tenant id.

https://github.com/folio-org/ui-marc-authorities/assets/77053927/8a0060cd-6571-49f3-abad-442de4327002

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Pull request merged**